__init__.py

```diff
@@ -0,0 +1,48 @@
+bl_info = {
+    "name": "Cesium Ion",
+    "description": "Cesium Ion Exporting Tool",
```

Comment:
Perhaps something like `Upload and tile models with Cesium ion. https://cesium.com` We always have to be mindful when communicating information to the user.  I assume this is just plain text?  (No markdown or anything?)